app/main.py

```diff
@@ -1,3 +1,58 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: (int, float)) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: (int, float)) -> object:
+        if not (
+            isinstance(other, Distance)
+            or isinstance(other, (int, float))
+        ):
+            raise TypeError(f"Cannot add {type(other)} to Distance")
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        return Distance(self.km + other)
```

Comment:
Use two same returns is not good.
Better to be to create an object for some variable and then return it) To keep one style
P.s. in previous comment I missed this statement  `self.km += other.km`, so you can use that likke you did before.